src/tests/functional/shared_test_classes/src/single_op/variadic_split.cpp

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/single_op/variadic_split.hpp"
+#include "ngraph_functions/builders.hpp"
```

Comment:
Remove #include "ngraph_functions/builders.hpp"
Please don't use builders unless it's necessary. We plan to remove most of them

Reply:
Thanks, done!